### ISSUE TITLE
Panel not available with Grafana v8.1.1

### ISSUE THREAD

**Issue by mattups** [marcusolsson/grafana-calendar-panel#28]
After successfully installing the plugin via CLI in Grafana v8.1.1 the plugin is listed as "Installed" but no panels are available to use inside dashboards

**Comment by SckyzO** [comment]
Hi team. 
Same problem for me. In version 7.3.6
No error in logs ... 
very strange...

**Comment by marcusolsson** [comment]
@SckyzO Did it reappear for you as well eventually?

**Comment by SckyzO** [comment]
Ho ... Yes ... Today it reappear ... w00t :) 
I don't know when and why. 
Nothing in logs ... very very strange.

**Comment by jayaraj** [comment]
Is this issue still open.

I get these errors while loading this plugin.

```
INFO[01-17|08:44:07] Request Completed                        logger=context userId=1 orgId=1 uname=admin method=GET path=/api/plugins/marcusolsson-calendar-panel/markdown/help status=404 remote_addr=[::1] time_ms=11 size=68 referer="http://localhost:3000/plugins/marcusolsson-calendar-panel?page=overview"
INFO[01-17|08:44:11] Downloaded marcusolsson-calendar-panel v0.5.0 zip successfully logger=plugin.installer
INFO[01-17|08:44:11] Plugin registered                        logger=plugin.manager pluginId=marcusolsson-calendar-panel
INFO[01-17|08:44:11] Request Completed                        logger=context userId=1 orgId=1 uname=admin method=GET path=/public/emotion status=404 remote_addr=[::1] time_ms=21 size=37403 referer="http://localhost:3000/plugins/marcusolsson-calendar-panel?page=overview"
```
<img width="1435" alt="Screenshot 2022-01-17 at 8 50 18 AM" src="https://user-images.githubusercontent.com/364636/149703102-c52631b2-7b22-41b0-9728-31d71e888f5a.png">

**Comment by jayaraj** [comment]
Do we need to update plugin with 
`import { css, cx } from '@emotion/css';` instead of `import { css, cx } from 'emotion';`

**Comment by marcusolsson** [comment]
Should be fixed in 0.6.0 and 0.7.0. Let me know if not, and I'll reopen the issue!
